modular_skyrat/modules/modular_weapons/code/modular_projectiles.dm

```diff
@@ -12,27 +12,6 @@
 *	.460 Ceres (renamed tgcode .45)
 */
 
-/obj/item/ammo_casing/c45/rubber
-	name = ".460 Ceres rubber bullet casing"
-	desc = "A .460 bullet casing.\
-	<br><br>\
-	<i>RUBBER: Less than lethal ammo. Deals both stamina damage and regular damage.</i>"
-	projectile_type = /obj/projectile/bullet/c45/rubber
-	harmful = FALSE
-
-/obj/projectile/bullet/c45/rubber
-	name = ".460 Ceres rubber bullet"
-	damage = 10
-	stamina = 30
-	ricochets_max = 6
-	ricochet_incidence_leeway = 0
-	ricochet_chance = 130
-	ricochet_decay_damage = 0.7
-	shrapnel_type = null
-	sharpness = NONE
-	embed_data = null
-	wound_bonus = -50
-
 /obj/item/ammo_casing/c45/hp
```

Comment:
What's your thinking for leaving some of the ceres HP variants? Naming may be odd if it not all calibers are consistent